src/index.js

```diff
@@ -13,7 +13,83 @@ import wrap from '@adobe/helix-shared-wrap';
 import { logger } from '@adobe/helix-universal-logger';
 import { wrap as status } from '@adobe/helix-status';
 import { Response } from '@adobe/fetch';
-import { GetObjectCommand, S3Client } from '@aws-sdk/client-s3';
+import {
+  DeleteObjectCommand, ListObjectsCommand, PutObjectCommand, CopyObjectCommand, S3Client,
+} from '@aws-sdk/client-s3';
+
+const VALID_MODES = ['preview', 'live'];
+const VALID_ACTIONS = ['store', 'evict'];
+
+const buildDefaultParams = () => ({
+  Bucket: 'franklin-content-bus-headless',
+});
+
+const deleteItemAction = (s3, key, deletedKeys) => new Promise((resolve, reject) => {
+  const params = buildDefaultParams();
+  params.Key = key;
```

Comment:
this is kind of a weird pattern. better:

```suggestion
  const params = buildDefaultParams({
    Key: key,
  });
```